R/tidy-upkeep.R

```diff
@@ -85,6 +85,20 @@ upkeep_checklist <- function(year = NULL) {
       ""
     )
   }
+if (year <= 2022) {
+    bullets <- c(bullets,
+     "2022",
+     "",
+     todo("`usethis::use_tidy_coc()`"),
+     todo("Update errors to rlang 1.0.0"),
```

Comment:
Is there a guide for this?

Reply:
I think @lionel- has this in progress, so we'll update once it's available

Reply:
There's https://rlang.r-lib.org/reference/topic-error-call.html and a guide about error chaining coming up.

Reply:
Here is the topic about error chaining: https://rlang.r-lib.org/reference/topic-error-chaining.html
And the one about cli formatting: https://rlang.r-lib.org/reference/topic-condition-formatting.html

Reply:
OK added.